### ISSUE TITLE
Bot wont connect. No error code

### ISSUE THREAD

**Issue by fyrebug72** [Firebottle/Firebot#180]
The bot worked fine during my stream. After I was done, I added a button to one of my scenes to allow the mods to change scenes. After I did this, the bot would not reconnect. I downloaded the newest version and it still would not connect. Deleted the new button and still no luck. What did I do wrong? Sorry to be a pain. JSON attached.

**Comment by fyrebug72** [comment]
I do
[Firebot General.zip](https://github.com/Firebottle/Firebot/files/1061258/Firebot.General.zip)
 not know if JSOn got in there so here it is again

**Comment by Firebottle** [comment]
Which button did you add? Did you try re-logging into the app?

**Comment by fyrebug72** [comment]
I added a scene change button but have gotten rid of it. And have tried
re-logging several times

**Comment by Firebottle** [comment]
What was the button name? I'm going to look and see if it's still in the json somewhere and just didnt get removed.

**Comment by fyrebug72** [comment]
Damn now I can not remember! lol It was on the Moderator scene. Maybe
general? UDK sorry!

**Comment by fyrebug72** [comment]
And yes I have tried restarting, No luck

**Comment by fyrebug72** [comment]
So I found the problem. I created a group that just included me because,
for some reason, when I monitor my stream on my laptop, I get put into the
Moderator scene, not the default. Well, this moring, like I was saying, I
was trying to make a scene change button. I added the Me group to both
scenes. that was what the problem was. I was trying to be in too many
scenes at the same time. Sorry for bothering you. But glad I got it fixed
